docs/contributing.rst

```diff
@@ -18,7 +18,7 @@ There are many ways to contribute to Scrapy. Here are some of them:
 * Report bugs and request features in the `issue tracker`_, trying to follow
   the guidelines detailed in `Reporting bugs`_ below.
 
-* Submit patches for new functionality and/or bug fixes. Please read
+* Submit patches for new functionalities and/or bug fixes. Please read
```

Comment:
I'm not a native speaker, but https://english.stackexchange.com/questions/196590/functionality-vs-functionalities-are-both-correct-and-idiomatic seems to suggest "functionality" is ok here

Reply:
Yea, either is fine I think. :)